src/components/queryExpressionEditor/expressionEditor/ExpressionEditorConnected.tsx

```diff
@@ -0,0 +1,45 @@
+import {connect} from 'react-redux';
+import * as _ from 'underscore';
+import {IReactVaporState} from '../../../ReactVapor';
+import {IReduxAction, ReduxUtils} from '../../../utils/ReduxUtils';
+import {ICheckboxActionPayload} from '../../checkbox/CheckboxActions';
+import {IDropdownOption} from '../../dropdownSearch/DropdownSearch';
+import {IDropdownSearchState} from '../../dropdownSearch/DropdownSearchReducers';
+import {IListBoxState} from '../../listBox/ListBoxReducers';
+import {FIELD_SELECT_ID} from '../fieldSelect/FieldSelect';
+import {OPERATOR_SELECT_ID} from '../operatorSelect/OperatorSelect';
+import {ExpressionEditor, IExpressionEditorDispatchProps, IExpressionEditorOwnProps, IExpressionEditorProps, IExpressionEditorStateProps} from './ExpressionEditor';
+import {removeExpressionEditor, updateExpressionEditor} from './ExpressionEditorActions';
+import {IExpressionEditorState} from './ExpressionEditorReducers';
+
+// TODO QUESTION R-V :
+// Is this the only way to get the selected option from a DropdownSearchConnected selector?
+// Note : Duplication of code with ValueSelectStringConnected.tsx

```

Comment:
we have `getDropdownSearchSelectedOption` in DropdownSearchUtils but that does not do exactly this.   

you could make a generic util in DropdownSearchUtils with getSelectedOption that would be useful eslewhere for sure. I would make it so that if there is no selected value that would return undefined.  

```
const getSelectedOptions = (options: IDropdownOptions[] = []): string => options.filter((option) => option.selected)[0];  
```